src/sg/gov/moh/healthcert/1.0/interim-healthcert-schema.json

```diff
@@ -0,0 +1,703 @@
+{
+  "$id": "https://schemata.openattestation.com/sg/gov/moh/healthcert/1.0/interim-healthcert-schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "Interim PDT or Vaccine HealthCert Schema",
+  "required": ["id", "name", "validFrom", "fhirVersion", "fhirBundle", "issuers", "$template", "notarisationMetadata"],
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Internal reference, usually serial number, of this document",
+      "examples": ["TEST001"]
+    },
+    "name": {
+      "type": "string",
+      "examples": ["HealthCert"]
+    },
+    "validFrom": {
+      "type": "string",
+      "format": "date-time",
+      "description": "Date from which the document is considered valid",
+      "examples": ["2021-04-27T03:18:51.972Z"]
+    },
+    "fhirVersion": {
+      "type": "string",
+      "examples": ["4.0.1"]
+    },
+    "fhirBundle": {
+      "type": "object",
+      "properties": {
+        "resourceType": { "const": "Bundle" },
+        "type": { "type": "string", "default": "collection" },
+        "entry": {
+          "type": "array",
+          "items": {
+            "anyOf": [
+              { "$ref": "#/definitions/Patient" },
+              { "$ref": "#/definitions/Specimen" },
+              { "$ref": "#/definitions/Observation" },
+              { "$ref": "#/definitions/Organisation" },
+              { "$ref": "#/definitions/Immunization" },
+              { "$ref": "#/definitions/ImmunizationRecommendation" },
+              { "$ref": "#/definitions/Location" }
+            ]
+          }
+        }
+      },
+      "required": ["resourceType", "type", "entry"],
+      "additionalProperties": false
+    },
+    "issuers": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": ["id", "name", "identityProof"],
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "revocation": {
+            "type": "object",
+            "properties": { "type": { "type": "string" } },
+            "examples": [{ "type": "NONE" }]
+          },
+          "name": {
+            "type": "string",
+            "examples": ["SAMPLE CLINIC"]
+          },
+          "identityProof": {
```

Comment:
open attestation supports a few types of identity proof, but we use DNS-DID now.
```
    "identityProofDnsDid": {
      "type": "object",
      "properties": {
        "type": {
          "type": "string",
          "enum": ["DNS-DID"]
        },
        "key": {
          "type": "string",
          "description": "Public key associated"
        },
        "location": {
          "type": "string",
          "description": "Url of the website referencing to document store"
        }
      },
      "required": ["type", "key", "location"]
    }
```

Reply:
Let me know if it should be `identityProofDnsDid` or just `identityProof`